baracoda/sql/schema.sql

```diff
@@ -47,4 +51,4 @@ CREATE TABLE alembic_version
 INSERT INTO alembic_version
     (version_num)
 VALUES
-    ('a32c725ae353');
+    ('e501839465f6');
```

Comment:
This doesn't appear to be pointing at the correct "head" version in the migrations.